definitions/v7/partis.yml

```diff
@@ -0,0 +1,189 @@
+    - { id: 27, cat: Console/Wii, desc: "Wii"}
+    - { id: 14, cat: Console/XBox, desc: "XboX"}
+    - { id: 49, cat: PC/Games, desc: "Mac Igre"}
+    - { id: 48, cat: PC/Games, desc: "Linux Igre"}
+    - { id: 64, cat: PC/Games, desc: "PC igre/Update & Patch"}
+    # Music / Glasba
+    - { id: 8, cat: Audio, desc: "Glasba/Ostalo"}
+    - { id: 47, cat: Audio/MP3, desc: "Glasba/Mp3"}
+    - { id: 46, cat: Audio/Lossless, desc: "Glasba/Flac"} # FLAC/WAV
+    - { id: 23, cat: Audio/Video, desc: "Videospoti"}
+    - { id: 22, cat: Audio/Video, desc: "Music DVD"}
+    # Programs / Programi
+    - { id: 15, cat: PC, desc: "PC programi/drugo"}
+    - { id: 16, cat: PC/ISO, desc: "PC programi/ISO"}
+    - { id: 58, cat: PC/Mac, desc: "Mac programi"}
+    - { id: 50, cat: PC, desc: "Linux programi"}
+    # XXX - not working - search only works for exactly categories cat=18,36,35,37,56,39 in this order
+    # - { id: 18, cat: , desc: "XXX-HD"}
+    # - { id: 36, cat: , desc: "XXX-XviD"}
+    # - { id: 35, cat: , desc: "XXX-Clip"}
+    # - { id: 37, cat: , desc: "XXX"}
+    # - { id: 39, cat: , desc: "XXX"}
+    # - { id: 56, cat: , desc: "XXX"}
+    # Other / Ostalo
+    - { id: 3, cat: Books/EBook, desc: "eKnjige"}
+    - { id: 19, cat: Other, desc: "Slike"}
+    - { id: 21, cat: Audio/Audiobook, desc: "AudioBook"}
+    - { id: 9, cat: PC/Mobile-Other, desc: "GBA"}
+    - { id: 25, cat: PC/Mobile-Other, desc: "GSM/Igre"}
+    - { id: 26, cat: PC/Mobile-Android, desc: "PDA"}
+    - { id: 29, cat: PC/Mobile-iOS, desc: "Ipod"}
+    - { id: 61, cat: PC/Mobile-Other, desc: "GSM/Programi"}
+    - { id: 62, cat: PC/Mobile-Other, desc: "GSM/Ostalo"}
+
+  modes:
+    search: [q]
+    tv-search: [q]
+    movie-search: [q]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+
+  - name: password
+    type: password
+    label: Password
+
+  # Broken on website for now
+  # - name: freeleech
+  #   type: checkbox
+  #   label: Search freeleech only
+  #   default: false
+
+  # Broken on website for now
+  # - name: drsi
+  #   type: checkbox
+  #   label: Search Drsi only
+  #   default: false
+
+  - name: sort
+    type: select
+    label: Sort results
+    default: 'seeders DESC'
+    options:
+      'seeders ASC': 'Seeders ascending' 
+      'seeders DESC': 'Seeders descending'
+      'leechers ASC': 'Leechers ascending' 
+      'leechers DESC': 'Leechers descending' 
+      'size ASC': 'Size ascending' 
+      'size DESC': 'Size descending' 
+
+login:
+  method: post
+
+  path: /user/login
+
+  inputs:
+    user: "{{ .Config.username }}"
+    pass: "{{ .Config.password }}"
+
+  error:
+    - selector: div[id=errLogin]
+
+  test:
+    path: /portal#prva
+    selector: a[id=pusername]
+
+search:
+  paths:
+    - path: /torrent/search
+  inputs:
+    q: "{{ .Keywords }}"
+    cat: "{{join .Categories \",\"}}"
+    order: "{{ .Config.sort }}"
+
+  preprocessingfilters:

```

Comment:
Html site that is scraping but returns json in the html?

Or can you use the existing Cardigann json support?

Reply:
Actual response is not "json" it's more like javascript - i.e. the actual response is
```js
data = {{"type":"text/plain; charset=utf-8","config":{"tpl":"browse"},"data":{},"values":{},"elements":{},"sets":{"torrent_list":{"cols":["fid","w","created_at","title","seeders","leechers","short_description","torrent_category_id","completed","size","icon","image"],"data":[
// The scraped data
], ... more junk}

window.onscroll = torrentScroll;

function arJs(){

}
```
Can't use json support as there is no way to clean up the response - remove text before and after the json array.